### PR TITLE
Add archivemount fuse-zip p7zip and libzip to Mariner

### PR DESCRIPTION
**Pull request by chalamalasetty**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 
Add archivemount fuse-zip p7zip and libzip packages to Mariner.
nmap-unix_crash.patch: Fix for ncat segmentation fault crash with unix sockets

###### Change Log  
  Add archivemount fuse-zip p7zip and libzip to Mariner
  nmap-unix_crash.patch: Fix for ncat segmentation fault crash with unix sockets

###### Does this affect the toolchain?  

YES
**NO**

###### Test Methodology
Local build.

### CODE REVIEW

File: SPECS/archivemount/archivemount.spec [hunk 1]
```diff
@@ -0,0 +1,171 @@
+Name:          archivemount
+Version:       0.9.1
+Release:       4%{?dist}
+Summary:       FUSE based filesystem for mounting compressed archives
+Vendor:        Microsoft Corporation
+Distribution:  Mariner
+License:       LGPLv2+
+URL:           http://www.cybernoia.de/software/archivemount/
```
Comment (thcrain-msft) [inline SPECS/archivemount/archivemount.spec:8]: Can we change these URLs to their HTTPS variants, since they seem to work?

**Review by thcrain-msft — APPROVED** [review]
LGTM, other than one nit on the archivemount URLs

File: SPECS/libzip/libzip.spec [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+%{_includedir}/zip.h
+%{_includedir}/zipconf*.h
+%{_libdir}/libzip.so
+%{_libdir}/pkgconfig/libzip.pc
+
+
+%changelog
+* Mon May 17 2021 Suresh Babu Chalamalasetty <schalam@microsoft.com> - 1.7.3-5
+- Initial CBL-Mariner import from Fedora 34 (license: MIT)
+- License verified
```
Comment (PawelWMS) [inline SPECS/libzip/libzip.spec:120]: Thank you for checking the licenses!

File: SPECS/p7zip/p7zip.spec [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+    DEST_BIN=%{_bindir} \
+    DEST_SHARE=%{_libexecdir}/p7zip \
+    DEST_MAN=%{_mandir}
+
+# remove redundant DOC dir
+mv %{buildroot}%{_docdir}/p7zip/DOC/* %{buildroot}%{_docdir}/p7zip
+rmdir %{buildroot}%{_docdir}/p7zip/DOC/
+
+%check
+%if ! 0%{?rhel} || 0%{?rhel} >= 7
```
Comment (PawelWMS) [inline SPECS/p7zip/p7zip.spec:104]: Nit: we can remove this check, it's always true for us.

**Review by PawelWMS — CHANGES_REQUESTED** [review]

**Review by PawelWMS — APPROVED** [review]

**Review by henryli001 — APPROVED** [review]

**Pull request merged**